syntax = "proto3";

service Matcher {
    rpc GetMatches(GetMatchesRequest) returns (GetMatchesReply) {}
    rpc RecalcScaNN(Empty) returns (Empty) {}
}

message GetMatchesRequest {
    string user_id = 1;
}

message GetMatchesReply {
    repeated string user_ids = 1;
}

message Empty {

}
